### PR TITLE
[DASH-2228] Add a Go Back button to hosting files tree view

### PR DESCRIPTION
**Pull request by powerczuk**
Tooltip style

screen shot 2016-10-07 at 11 15 49

Go back to the Hostings list view

screen shot 2016-10-10 at 13 29 55

### CODE REVIEW

File: src/apps/Hosting/Files.js [hunk 1]
```diff
@@ -86,7 +93,14 @@ const HostingFilesView = React.createClass({
         <HostingDialog />
         <HostingPublishDialog />
 
-        <InnerToolbar title="Website Hosting">
+        <InnerToolbar
+          title="Website Hosting"
+          style={{ padding: 0 }}
```
Comment (Flixow) [inline src/apps/Hosting/Files.js]: Is it necessary?

Comment (Flixow) [inline src/apps/Hosting/Files.js]: Is it necessary?

Comment (Flixow) [inline src/apps/Hosting/Files.js]: Is it necessary?

**Comment by Flixow** [comment]
LGTM

**Comment by wpasinski** [comment]
shouldn't we unify font sizes?

zrzut ekranu 2016-10-05 o 19 29 50
zrzut ekranu 2016-10-05 o 19 29 37

**Comment by patrykkopycinski** [comment]
@wpasinski 👍

**Comment by powerczuk** [comment]
screen shot 2016-10-06 at 13 53 34

File: src/common/InnerToolbar/InnerToolbar.js [hunk 2]
```diff
@@ -35,6 +35,11 @@ const InnerToolbar = Radium(React.createClass({
         flex: 1,
         justifyContent: 'flex-end'
       },
+      tooltip: {
```
Comment (patrykkopycinski) [inline src/common/InnerToolbar/InnerToolbar.js]: instead remove this line: https://github.com/Syncano/syncano-dashboard/blob/devel/src/common/InnerToolbar/InnerToolbar.js#L68
please read documentations.

**Comment by patrykkopycinski** [comment]
do you think it's ready?

**Comment by powerczuk** [comment]
Yes, now we have the same tooltips size in whole dashboard. Maybe I'm wrong, any suggestions?

**Comment by patrykkopycinski** [comment]
https://github.com/Syncano/syncano-dashboard/pull/2/files#diff-ff657fc4fac9ae602ededca41764333cR67
https://github.com/Syncano/syncano-dashboard/pull/2/files#diff-ff657fc4fac9ae602ededca41764333cR69
both lines are unnecessary

**Comment by powerczuk** [comment]
You're right, I forgot about that :/

**Comment by patrykkopycinski** [comment]
website hosting syncano dashboard

**Comment by patrykkopycinski** [comment]
@powerczuk new screenshot?

**Comment by powerczuk** [comment]
screen shot 2016-10-10 at 13 29 55

**Pull request merged**